wiki/Application-Issues.md

```diff
@@ -30,3 +30,19 @@ To fix this, run them with the `_JAVA_AWT_WM_NONREPARENTING=1` environment varia
 
 There's a bug in rofi-wayland that prevents it from accepting keyboard input on niri with errors in the output.
 It's been fixed in rofi, but the fix had not been released yet: https://github.com/davatorium/rofi/discussions/2008
+
+### Fullscreen games
+
+Many video games, both Linux-native and on Wine, can have all sorts of issues when using non-stacking desktop environments.
+Most of these can be avoided with Valve's [gamescope](https://github.com/ValveSoftware/gamescope), for example:
+
+```sh
+gamescope -f -w 1920 -h 1080 -W 1920 -H 1080 --force-grab-cursor --backend sdl -- <game>
```

Comment:
I reckon most people run games through Steam, where you go into launch options and enter something like `gamescope -f -w 1920 ... -- %command%`. Might be useful to add as a second example here?